test/mock/idam/services/getToken.js

```diff
@@ -1,7 +1,14 @@
+const jwt = require('jsonwebtoken');
+
+const token = jwt.sign({
+  data: 'eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiJocW02bHY5N2VtMG9nam5zNTgyZWFncHQyciIsInN1YiI6IjQxIiwiaWF0IjoxNTgwODk4Njg1LCJleHAiOjE1ODA5Mjc0ODUsImRhdGEiOiJjaXRpemVuLGNpdGl6ZW4scHJvYmF0ZS1wcml2YXRlLWJldGEsY2l0aXplbi1sb2ExLGNpdGl6ZW4tbG9hMSxwcm9iYXRlLXByaXZhdGUtYmV0YS1sb2ExIiwidHlwZSI6IkFDQ0VTUyIsImlkIjoiNDEiLCJmb3JlbmFtZSI6IkNpdGl6ZW4iLCJzdXJuYW1lIjoiQ2l0aXplbiIsImRlZmF1bHQtc2VydmljZSI6IlByb2JhdGUiLCJsb2EiOjEsImRlZmF1bHQtdXJsIjoiaHR0cHM6Ly9sb2NhbGhvc3Q6OTAwMC9wb2MvcHJvYmF0ZSIsImdyb3VwIjoicHJvYmF0ZS1wcml2YXRlLWJldGEifQ.RMz8QxChRnU4F1op6-NLOz4uf64aEpA-MfgFuU2oby4'
```

Comment:
data should include that data you want to include in the token, and the sign method would encrypt everything into a valid token with the expiry date, something like:

const token = jwt.sign({
  forename: 'John',
  surname: 'Smith'
}, 'secret', { expiresIn: '1h' });

This would produce a token including forename, surname and the expiration date, similar to the way Idam is producing the real token, and so we can extract the data in the overview controller